_includes/nav.njk

```diff
@@ -1,46 +1,48 @@
 <header class="header">
-    <a
-        href="/index.html" class="logo-section">
-        <!-- Mystery Code Society icon & name -->
-        <img src="/assets/images/icon/mcs-logo.svg">
-        <span class="brand-name-icon u-text-uppercase">
-            <span class="font-weight-bold">Mystery Code
+    <div class="container">
```

Comment:
This adds this container as a constraint.  (Container is used elsewhere to constrain max-width.